README.md

```diff
@@ -78,6 +78,29 @@ Create a new repository in Github and make a note of the URL of the new reposito
 
 Please read [the tutorial](./FUNCTIONS.md) on how to write custom functions for Google Sheets using Apps Script.
 
+## Testing your code
+
+You can run tests with jest using
+```
+npm run test
+```
+This has limitations:
+
+* You _can_ test code that has no dependencies to Google App Script code, e.g.
+```
+const hasCpuTime = () => !(Date.now() - START_TIME > ONE_MINUTE * 4);
+
+```
+* You _can not_ test code that has no dependencies to Google App Script code, e.g.
```

Comment:
Please change "has no dependencies" to "has dependencies"

Reply:
Done e2b6bb2

Reply:
Just stumbled upon [this repo](https://www.npmjs.com/package/gasmask) that generates stubs for testing Apps Script services locally.

Reply:
Interesting. I'll mention #9. Perhaps someone (me?) will have some time to try this out.